.github/workflows/contracts-test.yaml

```diff
@@ -21,7 +21,7 @@ jobs:
           cache: false
 
       - name: Cache Solidity ABI artifacts
-        uses: actions/cache@v2
+        uses: actions/cache@v4
```

Comment:
Why in this PR?

Reply:
Otherwise you get warnings / errors due to obsoletion